MotionRecognition/src/RecognizerController/CountNetworkTrainController.cs

```diff
@@ -108,7 +108,13 @@ public static void Train(ref NetworkContainer container, ref CountNetworkTrainSe
 				throw new IncorrectActionOrderException("Prepare network before training network.");
 
 			EncogWrapper.Train(ref container, ref settings.trainSettings);
-			EncogWrapper.SaveNetworkToFS(ref container, settings.outputDirectory + settings.outputName + ".eg");
+			if (settings.outputDirectory[settings.outputDirectory.Length - 1].Equals("/"))
+			{
+				EncogWrapper.SaveNetworkToFS(ref container, settings.outputDirectory + settings.outputName + ".eg");
```

Comment:
het liefst zie ik dit anders ipv een if en daarbuiten hetzelfde min '/' maar kijk maar of dit ez is of niet